python/pfs/drp/stella/constructFiberFlatTask.py

```diff
@@ -77,6 +83,22 @@ def combine(self, cache, struct, outputId):
         sumExpect = None  # Sum of what we expect
         for dd in dithers:
             image = coadds[dd]
+            self.log.info(f'Dither: {dd}')
```

Comment:
Don't use f-strings for logging. This should be:

    self.log.info("Dither: %s", dd)

Reply:
Fixed.